Examples/test-suite/python/director_throw_runme.py

```diff
@@ -0,0 +1,29 @@
+import director_throw
+
+class MyException(Exception):
+    pass
+
+go_called_count = 0
+class Derived(director_throw.Foo):
+    def go(self):
+      global go_called_count
+      go_called_count += 1
+      raise MyException
```

Comment:
Thanks for the autoconf snippet. It works except for a missing
AC_LANG_PUSH/AC_LANG_POP to switch to C++.

The exception specifications are not actually an issue even with
--fno-exceptions. On the other hand, playing with these tests made me
realize that there are autogenerated try-catch blocks that are a huge
problem. Going to have to look at those in more detail. -sigh-

Will also fix the indent.

On Thu, Mar 19, 2015 at 5:52 PM, Olly Betts <notifications@github.com>
wrote:

> In Examples/test-suite/python/director_throw_runme.py
> <https://github.com/swig/swig/pull/349#discussion_r26811930>:
>
> > @@ -0,0 +1,29 @@
> > +import director_throw
> > +
> > +class MyException(Exception):
> > +    pass
> > +
> > +go_called_count = 0
> > +class Derived(director_throw.Foo):
> > +    def go(self):
> > +      global go_called_count
> > +      go_called_count += 1
> > +      raise MyException
>
> Looks like a non-standard indent here still.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/swig/swig/pull/349/files#r26811930>.
>